e2e/test/prerequisites/iot_config.cmd_template

```diff
@@ -38,6 +27,20 @@ set IOTHUB_EVENTHUB_COMPATIBLE_NAME=
 @REM EventHub consumer group:
 set IOTHUB_EVENTHUB_CONSUMER_GROUP=
 
+@REM ---------- Secondary IoTHub configuration for running Polling Fault Injection tests ----------
```

Comment:
Why do we need this? Why not just run against the primary hub?
This makes it very difficult for 3rd parties to run our tests.

Reply:
Same as above. (https://github.com/Azure/azure-iot-sdk-csharp/pull/941#discussion_r291412800)